app/views/layouts/_header.html.erb

```diff
@@ -26,6 +26,9 @@
                 <li class="nav-item">
                     <%= link_to "Category", category_path, class: "nav-link" %>
                 </li>
+                    <li class="nav-item">
+                        <%= link_to "Manage Category", admin_path, class: "nav-link" %>
```

Comment:
I receive an error for Line 30.
![image](https://user-images.githubusercontent.com/35232348/96445846-2226a980-1243-11eb-9e11-cf5de20685bf.png)
**admin_path** does not seem to exist. Try a path for Admin Category Index.